tests/test_e2ee_utils.py

```
@@ -21,6 +22,24 @@
 )
 
 
+def _matrix_capabilities(encryption_available: bool = True):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_matrix_capabilities`

(ANN202)